lua/entities/gmod_wire_expression2/core/custom/cl_prop.lua

```diff
@@ -67,3 +67,8 @@ E2Helper.Descriptions["setPos(b:v)"] = "Sets the position of a bone."
 E2Helper.Descriptions["setAng(b:a)"] = "Set the rotation of a bone."
 E2Helper.Descriptions["ragdollSetPos(e:v)"] = "Sets the position of a ragdoll while preserving pose."
 E2Helper.Descriptions["ragdollSetAng(e:a)"] = "Set the rotation of a ragdoll while preserving pose."
+E2Helper.Descriptions["setEyeTarget"] = "For NPCs, sets the eye target to the world position. For ragdolls, sets the eye target to the local eye position"
```

Comment:
Yes, they work. I didn't know it was a hack. Admittedly I just find it easier (less typing) so I prefer using it.

Reply:
I don't see how ignoring the signature and only providing the name of a function wouldn't be a hack

Reply:
Well it's a good hack in my book because it lets me type less.

Reply:
What's it supposed to be? `setEyeTarget(e:)`? Is it okay without the args?

Reply:
Apparently so, I haven't checked myself. I thought that only worked for no parameter functions. Either way that behavior shouldn't be relied on, the signature should be `setEyeTarget(e:v)`.

Reply:
It's just a way for setting the description of multiple functions of the same name. It's not like someone's going to rewrite how E2Descriptions are handled anytime soon.

Reply:
This has nothing to do with rewriting the e2helper, that behavior just shouldn't be relied on since it may conflict with any other addon adding a function with that name.

Also you don't even need it for the usecase you gave, most of these functions don't have overloads.